### PR TITLE
api: port InsertRemoveSDCard to Rust 

### PR DESCRIPTION
**Pull request by benma**
Fixes https://github.com/digitalbitbox/bitbox-wallet-app/issues/436

### CODE REVIEW

**Comment by benma** [comment]
Builds on #591

File: src/workflow/backup.c [hunk 1]
```diff
@@ -81,11 +80,8 @@ bool workflow_backup_create(const CreateBackupRequest* request)
         return false;
     }
 
-    // Wait for sd card.
-    const InsertRemoveSDCardRequest sd = {
-        .action = InsertRemoveSDCardRequest_SDCardAction_INSERT_CARD,
-    };
-    sdcard_handle(&sd);
+    // Wait for sd card
+    rust_workflow_sdcard_blocking(true);
```
Comment (x1ddos) [inline src/workflow/backup.c:84]: Would be nice readability-wise to use an enum like before, `SDCARD_ACTION_INSERT` and `SDCARD_ACTION_REMOVE`.

**Review by x1ddos — APPROVED** [review]
LGTM. Nice.

File: src/rust/bitbox02/src/ui/ui_stub.rs [hunk 2]
```diff
@@ -75,6 +75,13 @@ where
     panic!("not implemented")
 }
 
+pub fn sdcard_create<'a, F>(_insert: bool, _continue_callback: F) -> Component<'a>
+where
+    F: FnMut() + 'a,
+{
+    panic!("not implemented")
```
Comment (x1ddos) [inline src/rust/bitbox02/src/ui/ui_stub.rs:82]: Forgot: wouldn've been also nice to make a test with this.

Comment (benma) [reply src/rust/bitbox02/src/ui/ui_stub.rs]: done, PTAL

Comment (x1ddos) [reply src/rust/bitbox02/src/ui/ui_stub.rs]: :+1: thanks!

**Pull request merged**